src/test/docker/travis/Dockerfile

```diff
@@ -48,7 +48,11 @@ RUN case $BASE_IMAGE in \
     esac
 
 # Install extra dependencies if necessary here.
-#
+RUN case $BASE_IMAGE in \
+     bionic*) apt-get update -y && apt-get install -y libs3-dev ;; \
```

Comment:
I'll update our `fluxrm/testenv` images in a separate PR and then  you can drop this line.